Content.Shared/Atmos/EntitySystems/SharedGasPressurePumpSystem.cs

```diff
@@ -0,0 +1,97 @@
+using Content.Shared.Administration.Logs;
+using Content.Shared.Atmos.Components;
+using Content.Shared.Atmos.Piping;
+using Content.Shared.Atmos.Piping.Binary.Components;
+using Content.Shared.Atmos.Piping.Components;
+using Content.Shared.Database;
+using Content.Shared.Examine;
+using Content.Shared.Popups;
+using Content.Shared.Power;
+using Content.Shared.Power.Components;
+using Content.Shared.Power.EntitySystems;
+using Content.Shared.UserInterface;
+
+namespace Content.Shared.Atmos.EntitySystems;
+
+public abstract class SharedGasPressurePumpSystem : EntitySystem
+{
+    [Dependency] private   readonly ISharedAdminLogManager _adminLogger = default!;
+    [Dependency] protected readonly SharedAppearanceSystem Appearance = default!;
+    [Dependency] private   readonly SharedPowerReceiverSystem _receiver = default!;
+    [Dependency] protected readonly SharedUserInterfaceSystem UserInterfaceSystem = default!;
```

Comment:
Huh, when did we start aligning variable modifiers like this?

Reply:
I do it but I think it's just a me thing.